src/styles/LESS/detail-page.less

```diff
@@ -1,25 +1,46 @@
 .city-page {
-    display: flex;
-    justify-content: center;
-    .background-image("/compare_screen_full.jpg");
+  display: flex;
+  justify-content: center;
+  .background-image("/compare_screen_full.jpg");
 }
 
 .city-details {
-    border: 1px solid @white;
-    width: 80%;
-    margin: 3.5% 0 3.5% 0;
-    border-radius: 2px;
-    // background: @white;
-    background: transparent;
-
+  border: 1px solid @white;
+  width: 80%;
+  margin: 3.5% 0 3.5% 0;
+  border-radius: 2px;
+  // background: @white;
+  background: transparent;
```

Comment:
Sorry this hasn't been updated yet, the PR should be merged soon and we can handle any merge conflicts in it